src/sql/workbench/parts/notebook/browser/models/cell.ts

```diff
@@ -94,6 +102,34 @@ export class CellModel implements ICellModel {
 		return this._future;
 	}
 
+	public get isCollapsed() {
+		return this._isCollapsed;
+	}
+
+	public set isCollapsed(value: boolean) {
+		this._isCollapsed = value;
+
+		let tagIndex = -1;
+		if (Array.isArray(this._metadata.tags)) {
+			tagIndex = this._metadata.tags.findIndex(tag => tag === this._hideInputTag);
+		}
```

Comment:
if it's not an array, what happens?

Reply:
It's explicitly an array type. I'll switch this to a null check.